src/otx/recipe/detection/rtdetr_101.yaml

```diff
@@ -58,26 +58,36 @@ overrides:
       batch_size: 4
       to_tv_image: true
       transforms:
-        - class_path: torchvision.transforms.v2.RandomPhotometricDistort
-          init_args:
-            p: 0.5
         - class_path: torchvision.transforms.v2.RandomZoomOut
           init_args:
             fill: 0
-        - class_path: otx.core.data.transform_libs.torchvision.RandomFlip
```

Comment:
Why was it deleted?

Reply:
It wasn't deleted, it was moved to L74 to minimize the number of numpy (otx augs) <-> tvtensor (torchvision augs) conversions